x-pack/plugins/uptime/public/components/fleet_package/browser/advanced_fields.tsx

```diff
@@ -39,6 +41,73 @@ export const BrowserAdvancedFields = () => {
       data-test-subj="syntheticsBrowserAdvancedFieldsAccordion"
     >
       <EuiSpacer size="m" />
+      <EuiDescribedFormGroup
+        title={
+          <h4>
+            <FormattedMessage
+              id="xpack.uptime.createPackagePolicy.stepConfigure.browserAdvancedSettings.filtering.title"
+              defaultMessage="Synthetics agent filtering"
+            />
+          </h4>
+        }
+        description={
+          <FormattedMessage
+            id="xpack.uptime.createPackagePolicy.stepConfigure.browserAdvancedSettings.filtering.description"
+            defaultMessage="Filter tests using match or tags options."
+          />
+        }
+      >
+        <EuiSpacer size="s" />
+        <EuiFormRow
+          label={
+            <FormattedMessage
+              id="xpack.uptime.createPackagePolicy.stepConfigure.browserAdvancedSettings.journeyFiltersMatch.label"
+              defaultMessage="Filter match"
+            />
+          }
+          labelAppend={<OptionalLabel />}
+          helpText={
+            <FormattedMessage
+              id="xpack.uptime.createPackagePolicy.stepConfigure.browserAdvancedSettings.journeyFiltersMatch.helpText"
+              defaultMessage="Run only journeys with a name or tags that matches the provided glob."
```

Comment:
```suggestion
              defaultMessage="Run only journeys with a name or tag that matches the provided glob."
```

Think this is cleaner copy.